### PR TITLE
Add Uniforms Component

### PR DESCRIPTION
**Pull request by Mike-MF**
**When merged this pull request will:**
- Adds PMC Casual Uniforms, more will come as/when Mick gets ideas.

### CODE REVIEW

File: addons/uniforms/CfgVehicles.hpp [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    class CLASS(Uniform_PMC_Casual_Arcadian_V): C_Uniform_ArtTShirt_01_v1_F {
+        scope = 2;
+        author = "GilleeDoo, Mike";
+        uniformClass = QCLASS(Uniform_PMC_Casual_Arcadian);
+        hiddenSelectionsTextures[] = {QPATHTOF(data\pmc_casual_arcadian_co.paa)};
+    };
+    // GT Armoury
+    class CLASS(Uniform_PMC_Casual_GTArmoury_V_Item): Item_Base_F {
+        displayName = "PMC Casual (GT Armoury)";
```
Comment (jonpas) [inline addons/uniforms/CfgVehicles.hpp:42]: But it's "Armory".

**Comment by jonpas** [comment]
Why not Services?

**Comment by Mike-MF** [comment]
Specific to us only.

File: addons/uniforms/CfgVehicles.hpp (hunk 1 above)
Comment (Mike-MF) [reply addons/uniforms/CfgVehicles.hpp:42]: The image has "Armoury" in it now

**Pull request merged**